core/src/main/scala/org/apache/spark/deploy/Client.scala

```diff
@@ -125,10 +126,12 @@ private class ClientActor(driverArgs: ClientArguments, conf: SparkConf)
         }
         System.exit(0)
     }
+    // scalastyle:on println
   }
 
   override def receiveWithLogging: PartialFunction[Any, Unit] = {
 
+    // scalastyle:off println
     case SubmitDriverResponse(success, driverId, message) =>
       println(message)
```

Comment:
Any guidelines on what should be a println or a log?
I'll be glad to go through and check them.